src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure(); // generation random figures for array
+        }
+
+        // generation default figure for array
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the beginning of the class definition.